test/models/User/Repository.test.ts

```diff
@@ -0,0 +1,62 @@
+import Database from "../../../src/config/Database";
+import { UserRepository } from "../../../src/models/User/Repository";
+import { User } from "../../../src/models/User";
+
+describe("UserRepository", () => {
+  beforeAll(async () => {
+    await Database.setConnection();
+  });
+
+  beforeEach(async () => {
+    await User.destroy({ truncate: true });
+  });
+
+  afterAll(async () => {
+    await Database.close();
+  });
+
+  it("creates a user", async () => {
+    const email = "asd@qwe.com";
+    const user = await UserRepository.create({ email: email, password: "AValidPassword123" });
+
+    expect(user.id).not.toBeNull();
+    expect(user.email).toEqual(email);
+  });
+
+  it("delegates password validation to the model", async () => {
+    await expect(UserRepository.create({
+      email: "asd@qwe.com",
+      password: "an invalid password"
+    })).rejects.toThrow();
+  });
+
+  it("delegates email validation to the model", async () => {
+    await expect(UserRepository.create({
+      email: "asd@qwecom",
+      password: "AValidPassword123"
+    })).rejects.toThrow();
+  });
+
+  it("finds a user by email", async () => {
```

Comment:
No entiendo porque creaste 2 users para hacer el find

Reply:
Para asegurarte que encuentre el correcto?

Reply:
Supongo que lo hizo por eso si jajaj

Reply:
Lo dejo asi? Osea no viene mal porque testea que encuentre el correcto cuando hay mas de uno. Quizas le cambio el nombre a la descricion del it no?

Reply:
dale